test/stores/file-store-test.js

```diff
@@ -128,6 +129,54 @@ vows.describe('nconf/stores/file').addBatch({
         assert.isNull(err);
         assert.deepEqual(read, data);
       }
+    },
+    "the saveToFile() method": {
+      topic: function (tmpStore) {
+        var that = this,
+            pathFile = '/tmp/nconf-save-toFile.json';
+
+        Object.keys(data).forEach(function (key) {
+          tmpStore.set(key, data[key]);
+        });
+
+        tmpStore.saveToFile(pathFile, function () {
+          fs.readFile(pathFile, function (err, d) {
+            fs.unlinkSync(pathFile);
+
+            return err
+              ? that.callback(err)
+              : that.callback(err, JSON.parse(d.toString()));
+          });
+        });
+      },
+      "should save the data correctly": function (err, read) {
+        assert.isNull(err);
+        assert.deepEqual(read, data);
+      }
+    },
+    "the saveToFile() method with custom format": {
+      topic: function (tmpStore) {
+        var that = this,
+            pathFile = '/tmp/nconf-save-toFile.yaml';
+
+        Object.keys(data).forEach(function (key) {
+          tmpStore.set(key, data[key]);
+        });
+
+        tmpStore.saveToFile(pathFile, yamlFormat, function () {
+          fs.readFile(pathFile, function (err, d) {
+            //fs.unlinkSync(pathFile);
```

Comment:
Can you remove this line and the next? I assume it's probably left there from some debugging :-)

Reply:
in fact what I did for debugging was comment the fs.unlink

Reply:
voilà